### DISCUSSION TITLE
Kubernetes(k3s) Identifying Homematic RF-Hardware: cat: can't open '/sys/class/net/eth0/address': No such file or directory

### DISCUSSION THREAD

**Discussion by MrConjurer** [OpenCCU/OpenCCU#2635] [🇩🇪 Fragen&Antworten] [answered]
Hallo zusammen,

ich habe RaspberryMatic mit dem Helm Chart auf Kubernetes(k3s) laufen. Das Betriebssystem ist Ubuntu.
Beim Starten des POD kommt die folgende Fehlermeldung.
Identifying Homematic RF-Hardware: cat: can't open '/sys/class/net/eth0/address': No such file or directory
In dem Alarmmeldungen bekomme ich immer folgende zwei Meldungen:
WatchDog: irqbalance-restart
WatchDog: no-internet
In den Netzwerkeinstellungen ist auch keine IP usw. zu sehen.
Beim Versuch ein Gerät per CUxD einzubinden bin ich ebenfalls gescheitert. 
Ich habe das Gefühl, dass ich etwas noch nicht richtig konfiguriert habe.
Hat jemand einen Tipp?

Lieben Dank.

Gruß
Marcel

**Comment by MrConjurer** [comment]
Hallo zusammen, hat sich schon erledigt bei mir heißt der Netzwerkadapter nicht eth0.

**Reply by jens-maus** [reply]
Und die Lösung war?

**Comment by MrConjurer** [comment, marked as answer]
Ich habe im netplan vom Ubuntu den Namen vom Netzwerkadapter in eth0 geändert.
